frontend/src/components/layout/Sidepanel.vue

```diff
@@ -93,7 +93,13 @@
                 {
                     icon: '/assets/icons/speed.svg',
                     text: 'Speed Overview',
-                    route: 'speedoverview'
+                    route: 'page-timings',
```

Comment:
This key probably redundant, since sidenav not using it when sublinks are defined